scripts/overall_stats.sql

```diff
@@ -0,0 +1,99 @@
+	ELSE sr.server_base END) as 'Servers' 
+FROM `observation_loinc` ol
+	join loinc l on l.id = ol.`loinc`
+	join stats_run_observation sro on sro.id = ol.observation
+	join stats_run_patient srp on srp.id = sro.patient
+	join stats_run sr on srp.`stats_run` = sr.id
+group by ol.loinc
+order by count(distinct ol.id) desc, code;
+
+/* Summary of Successes by Method */
+select group_concat(distinct CASE 
+	WHEN sr.`server_base`='http://hapi.fhir.org/baseDstu3' THEN 'HAPI3' 
+	WHEN sr.`server_base`='http://hapi.fhir.org/baseDstu2' THEN 'HAPI2' 
+	WHEN sr.`server_base`='https://syntheticmass.mitre.org/fhir' THEN 'MITRE' 
+	WHEN sr.`server_base`='https://r3.smarthealthit.org' THEN 'R3' 
+	WHEN sr.`server_base`='https://r2.smarthealthit.org' THEN 'R2' 
+	WHEN sr.`server_base`='https://open-ic.epic.com/FHIR/api/FHIR/DSTU2' THEN 'EPIC' 
+	ELSE sr.server_base END) as 'Servers', 
+count(r.id) as 'Count', m.description as 'Method' from method_result r
+	inner join method_type m on r.`method_type` = m.id
+	inner join stats_run_observation o on r.observation = o.id
+	inner join stats_run_patient p on p.id = o.patient
+	inner join stats_run sr on p.`stats_run` = sr.id
+where hpo_term_id is not null group by method_type;
+
+/* Conversion Failures by Exception Type */
+select group_concat(distinct CASE 
+	WHEN sr.`server_base`='http://hapi.fhir.org/baseDstu3' THEN 'HAPI3' 
+	WHEN sr.`server_base`='http://hapi.fhir.org/baseDstu2' THEN 'HAPI2' 
+	WHEN sr.`server_base`='https://syntheticmass.mitre.org/fhir' THEN 'MITRE' 
+	WHEN sr.`server_base`='https://r3.smarthealthit.org' THEN 'R3' 
+	WHEN sr.`server_base`='https://r2.smarthealthit.org' THEN 'R2' 
+	WHEN sr.`server_base`='https://open-ic.epic.com/FHIR/api/FHIR/DSTU2' THEN 'EPIC' 
+	ELSE sr.server_base END) as 'Servers', 
+count(o.id) as 'Count', e.`description` as 'Exception' from stats_run_observation o
+	join exception_type e on o.exception_type = e.id
+	inner join stats_run_patient p on p.id = o.patient
+	inner join stats_run sr on p.`stats_run` = sr.id
+group by o.exception_type;
+
+/* Method Failures by Exception Type */
+select group_concat(distinct CASE 
+	WHEN sr.`server_base`='http://hapi.fhir.org/baseDstu3' THEN 'HAPI3' 
+	WHEN sr.`server_base`='http://hapi.fhir.org/baseDstu2' THEN 'HAPI2' 
+	WHEN sr.`server_base`='https://syntheticmass.mitre.org/fhir' THEN 'MITRE' 
+	WHEN sr.`server_base`='https://r3.smarthealthit.org' THEN 'R3' 
+	WHEN sr.`server_base`='https://r2.smarthealthit.org' THEN 'R2' 
+	WHEN sr.`server_base`='https://open-ic.epic.com/FHIR/api/FHIR/DSTU2' THEN 'EPIC' 
+	ELSE sr.server_base END) as 'Servers', 
+count(r.id) as 'Count', e.`description` as 'Exception' from method_result r
+	join exception_type e on r.exception_type = e.id
+	inner join stats_run_observation o on r.observation = o.id
+	inner join stats_run_patient p on p.id = o.patient
+	inner join stats_run sr on p.`stats_run` = sr.id
+group by r.exception_type;
```

Comment:
May also want to add the `e.description` to group by statement, just to make sure you don't have any ambiguity in the group by statement.